js/src/components/google-combo-account-card/index.scss

```diff
@@ -0,0 +1,10 @@
+.google-combo-account-card {
+	.gla-account-card__description {
+		.components-checkbox-control {
+			display: inline-block;
+		}
+	}
+	p {
+		color: #757575;
```

Comment:
```suggestion
		color: $gray-700;
```

Reply:
This variable is available in the following file:
https://github.com/woocommerce/google-listings-and-ads/blob/develop/js/src/css/abstracts/_colors.scss#L2

which equals to #757575 https://github.com/WordPress/gutenberg/blob/33e2fadeebe32a64a946106488120b6f559d3deb/packages/base-styles/_colors.scss#L11